completions/_abbr

```diff
@@ -10,197 +10,196 @@
 #
 # ------------------------------------------------------------------------------
 
+_abbr (){
+  local state line
 
-local state line
+  local -i ret
 
-local -i ret
+  ret=1
 
-ret=1
-
-_arguments -C \
+  _arguments -C \
+    '--help[Show the manpage.]' \
+    '(-v --version)'{-v,--version}'[Show the current version.]' \
```

Comment:
Gotcha, I will revert it.

Since `_values` has a syntax similar to `_arguments`, we could group the related commands the same way, but this may be unnecessary?

Reply:
> Since _values has a similar syntax to _arguments, we could group the related commands the same way, but this may be unnecessary

Say more?

Reply:
Sure, an example would be:

```shell
{a,add}"[Add a new abbreviation.]" \
{c,clear-session}"[Erase all session abbreviations.]" \
# ...
{R,rename}"[Rename an abbreviation.]" \
{version,-v,--version}"[Show the current version.]"
```

In this case I don't think it would ne needed to use the parentheses syntax, i.e.

```shell
"(R rename)"{R,rename}"[Rename an abbreviation.]"
```

for excluding related commands/options, since in this context only one can be used and suggested.

Reply:
I think we can click resolve on this. If you agree @Farid-NL, go ahead and do it